airbyte-protocol/models/src/main/resources/protocol_models/airbyte_schema.yaml

```diff
@@ -0,0 +1,58 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/models/src/main/resources/protocol_models/airbyte_schema.yaml
+title: AirbyteSchema
+description: Airbyte schema type definition
+type: object
+additionalProperties: false
+required:
+- streams
+properties:
+  streams:
+    type: array
+    items:
+      "$ref": airbyte_schema.yaml#/definitions/AirbyteStream
+  metadata:
+    "$ref": airbyte_metadata.yaml#/definitions/AirbyteMetadata
+
+definitions:
+
+  AirbyteStream:
+    type: object
+    additionalProperties: false
+    required:
+    - id
+    - name
+    - fields
+    properties:
+      id:
```

Comment:
do we need this? i know singer has it. i'm sure we'll eventually encounter why they added it, but until now, it's not super clear what we'd use this for.